Update troubleshooting-dataflow-issues-connection-to-the-data-source.md (#438)
MicrosoftDocs/powerquery-docs: powerquery-docs/dataflows/troubleshooting-dataflow-issues-connection-to-the-data-source.md

   ![Gateway selector.](media/troubleshooting-dataflow-issues-connection-to-the-data-source/gateway-selection.png)

1. After you apply the change by selecting **OK**, repeat steps 1 through 4 to select the original gateway again.

These steps essentially delete all the data source bindings for the dataflow. After finishing these steps, you might be asked to set up credentials. When you've set up the gateway and your credentials successfully, you effectively "trimmed" the data source bindings for the dataflow to just the ones that the dataflow is actually using.

> Note: 
> For dataflows with cloud data sources without on-premises data gateway, when the datasource name changes, there would be an old data source connection still existing which can be removed by going to Manage Connections and Gateways page > Connections > Remove the old/unused data source connection and then go to the Dataflow > Edit Tables and then click Save & Close for the Dataflows to be updated to current data source connection and remove older connection 

Reviewer: ```suggestion
> [!NOTE] 
> For dataflows with cloud data sources without on-premises data gateway, when the data source name changes, an old data source connection still exists. You can remove this connection by going to the Manage Connections and Gateways page > Connections > Remove the old/unused data source connection. Then go to the Dataflow > Edit Tables and select **Save & Close** for the Dataflows to be updated to the current data source connection and to remove the older connection. 

```
